code/modules/cargo/exports/large_objects.dm

```diff
@@ -171,13 +171,13 @@
 	var/worth = 10
 	var/gases = C.air_contents.gases
 
-	worth += gases[/datum/gas/bz]*4
-	worth += gases[/datum/gas/stimulum]*25
-	worth += gases[/datum/gas/hypernoblium]*1000
-	worth += gases[/datum/gas/miasma]*4
-	worth += gases[/datum/gas/tritium]*7
-	worth += gases[/datum/gas/pluoxium]*6
-	worth += gases[/datum/gas/nitryl]*30
+	worth += gases[/datum/gas/bz]*2
+	worth += gases[/datum/gas/stimulum]*20
+	worth += gases[/datum/gas/hypernoblium]*100
```

Comment:
No need to nerf these two, they are hard to make.

Reply:
This is way to powerfull to make and sell.